recipes/setup_server.rb

```diff
@@ -17,4 +17,23 @@
 
 include_recipe "abiquo::setup_ui"
 include_recipe "abiquo::setup_websockify"
+
+template '/opt/abiquo/tomcat/conf/Catalina/localhost/api.xml' do
+    source 'api-m.xml.erb'
+    owner 'root'
+    group 'root'
+    variables (lazy {{ :db => node['abiquo']['db'] }})
```

Comment:
Also, if there is nothing to resolve here, what is the benefit of passing a db variable? At least to me, it is clearer to have the full node object in the template. Reading the template is much easier to understand.